pos_session_report/models/pos_session.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# © 2016 KMEE INFORMATICA LTDA (<http://kmee.com.br>)
+# Hendrix Costa <hendrix.costa@kmee.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields
+
+
+class PosSession(models.Model):
+    _inherit = 'pos.session'
+
+    user_id = fields.Many2one('res.users',
```

Comment:
why do you redefined this point_of_sale field ? 
https://github.com/odoo/odoo/blob/8.0/addons/point_of_sale/point_of_sale.py#L268